codeowners_diff.py

```diff
@@ -106,10 +106,8 @@ def _path_to_glob(path: str) -> str:
 
 def find_affected_files(path: str, repo: GitRepo) -> frozenset[str]:
     glob_ = _path_to_glob(path)
-    paths = glob.glob(
-        glob_, root_dir=repo.root_dir, recursive=True,
-    )
-    return repo.ls_files(paths)
+    paths = glob.glob(glob_, root_dir=repo.root_dir, recursive=True)
+    return repo.ls_files(paths) if paths else frozenset()
```

Comment:
If the problem is that `repo.ls_files` returns all the files in the repo if we give it an empty sequence, that sounds like a problem with that method -- we should probably fix it there.

I assume it's out of hours for you now so I'll make that change separately so I can cut a release tonight.

Reply:
No worries. I was in a rush earlier and brain couldn't connect that because `collections.abc.Sequence` required `__len__` that we can use a simple truthiness check inside `ls_files()` where the type is `Sequence[str]`. Thanks for tidying it up.